ballista/rust/executor/src/cpu_bound_executor.rs

```diff
@@ -311,6 +311,8 @@ mod tests {
     }
 
     #[tokio::test]
+    #[ignore]
+    // related https://github.com/apache/arrow-datafusion/issues/2140
```

Comment:
This test appears to have been introduced in https://github.com/apache/arrow-datafusion/pull/1883

Reply:
Yes, i will check later. Maybe it related to `tokio`